source/01-EQ/01.ptx

```diff
@@ -111,8 +132,62 @@
   </task>
   \<task>
       <p>
-        Draw the solution to the inequality on a number line.
-      </p>
+        Draw the solution to the inequality on a number line. 
+      <ol marker="A." cols="1">
+        <li><p> 
+          <image>
+              <latex-image>
+                \begin{tikzpicture}[scale=7]
```

Comment:
@StevenClontz  Is there a way we should make these number line images in sageplot, or should we just stick with tikz here?

Reply:
@siwelwerd  @StevenClontz also, I need help with two parts of this... how do I get the < end of the arrow on the left side?
For part (d) i need it to be [ on the 2 but it won't include that for me - it keeps opening stuff....

Reply:
If you do `\draw[<-, thick]` it will put an arrow on the left but not the right, or `\draw[<->, thick]` to get arrows on both ends

Reply:
I don't think I understand your question about part (d) though

Reply:
it looks like answer choice (d) is displaying correctly in the html version, it was showing me a ( instead of [ on code chat.